ouroboros-network-framework/src/Ouroboros/Network/ConnectionManager/Types.hs

```diff
@@ -79,6 +79,10 @@
 -- Termination procedure as well as connection state machine is not described in
 -- this haddock, see associated specification.
 --
+
+-- GR-FIXME[D]: references to PeerStateActions and P2P Governor feel misplaced
+-- as these are defined in a calling package! (DOU)
+
```

Comment:
I hoped that this will make the larger picture clearer.  I will replace this part of a diagram with a sentence which explains it.

Reply:
Hmm, another thought, maybe the governor is in the wrong package and we should move it to `ouroboros-network-framework`.